emloop_tensorflow/model.py

```diff
@@ -44,7 +45,7 @@ def __init__(self,  # pylint: disable=too-many-arguments
                  dataset: Optional[el.AbstractDataset], log_dir: Optional[str], inputs: List[str], outputs: List[str],
```

Comment:
I would make the `log_dir` indeed optional with default `None`.

Reply:
Is this change you propose in some way related to this pull request?